Dockerfile

```
@@ -10,9 +10,12 @@ RUN unzip ot-platform-api-latest.zip
 
 RUN chmod +x ot-platform-api-latest/bin/ot-platform-api
 
+ENV JVM_XMS=2g
+ENV JVM_XMX=7g
+
 ENTRYPOINT ["bash", "-c", "ot-platform-api-latest/bin/ot-platform-api \
-    -J-Xms2g \
-    -J-Xmx7g \
+    -J-Xms${JVM_XMS} \
+    -J-Xmx${JVM_XMX} \
```

```suggestion
    -J-Xms${JVM_XMS:-2g} \
    -J-Xmx${JVM_XMX:-7g} \
```

I think it is not needed to provide `ENV` statements for the defaults, as we won't use them at build time.